### PR TITLE
use variables from color settings

### PR DESCRIPTION
**Pull request by darkthemer**
- replaces most hardcoded color values with variables found in color settings
- adds `--brightblue` color variable

### CODE REVIEW

File: Steam.css [hunk 1]
```diff
@@ -326,6 +326,21 @@ img._24_AuLm54JVe1Zc0AApCDR._3d_bT685lnWotXxgzKW6am.yDr03475kalWBTwAE-Rnw {
...
+    color: var(--darkerwhite) !important;
+}
+
+._2-O4ZG0KrnSrzISHBKctFQ._1LVQp3h_vGOZ8TuDUEge0k {
+    color: var(--brightblue) !important;
+}
+
+._2-O4ZG0KrnSrzISHBKctFQ ._1vO6BoiVslZgs1kqDGdUs8 ._3uk3F1BmAJQR6rl1TPcaif {
+    color: var(--blu) !important;
+}
```
Comment (ShadowMonster99) [inline Steam.css:343]: typo in the word "blue", and what does this do?

File: Steam.css [hunk 2]
```diff
@@ -414,6 +450,22 @@ img._24_AuLm54JVe1Zc0AApCDR._3d_bT685lnWotXxgzKW6am.yDr03475kalWBTwAE-Rnw {
...
+    background: var(--brightblue) !important;
+}
+
+._2UyOBeiSdBayaFdRa39N2O:hover:not(._2hcblpUHuN5CYOoq6eU2y0), ._7AlhCx3XGzBeIrQaCneUD:hover:not(._2cO0HdvLEGAdhosLekWsmo) ._19axKcqYRuaJ8vdYKYmtTQ {
+    color: var(--white) !important;
+}
+
+._2UyOBeiSdBayaFdRa39N2O, ._7AlhCx3XGzBeIrQaCneUD ._19axKcqYRuaJ8vdYKYmtTQ {
+    color: var(--darkwhite) !important;
+}
```
Comment (ShadowMonster99) [inline Steam.css:468]: What does this add to the theme?

File: steam/Properties.css [hunk 3]
```diff
@@ -19,6 +19,39 @@
...
+
+._3qIXY674NedSZNa-CXtNlt, ._2OJfkxlD3X9p8Ygu1vR7Lr {
+    color: var(--darkerwhite) !important;
+}
+
+.bkfjn0yka2uHNqEvWZaTJ:not(.Myra7iGjzCdMPzitboVfh):hover {
+    background-color: var(--color-dark) !important;
+    color: var(--white) !important;
+}
+
```
Comment (ShadowMonster99) [inline steam/Properties.css:54]: What does this add?

File: Steam.css (hunk 1 above)
Comment (darkthemer) [reply Steam.css:343]: "blu" is assigned to #67c1f5

this applies the color variables to the library left sidebar's text

File: Steam.css (hunk 2 above)
Comment (darkthemer) [reply Steam.css:468]: color variables for header text (store, library, etc)

File: steam/Properties.css (hunk 3 above)
Comment (darkthemer) [reply steam/Properties.css:54]: applies color variables to headings, texts, subtexts, switches, sliders, dropdown menu arrows in the settings window + some other pop-ups/dialogs

**Pull request merged**